client/room/LifeInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import OpenLayout from 'client/room/OpenLayout';
+import ScoreInfoField from 'client/room/ScoreInfoField';
+
+function LifeInfo() {
+  const [open, setOpen] = useState(false);
+
+  const arr = [
+    '의사소통',
+    '위치 및 교통편',
+    '가격대비 만족도',
+    '청결도',
+    '방음',
+    '편의시설',
+  ];
+
+  return (
+    <OpenLayout open={open} setOpen={setOpen} title="생활 정보">
+      {arr.map((item, index) => (
+        <ScoreInfoField
```

Comment:
이 arr가 상수라면 컴포넌트가 리렌더링될때 다시 리스트를 만들지 않도록 컴포넌트 밖으로 분리하는게 좋을것 같습니다. 
그리고 리스트 이름은 어떤 리스트인지 알아볼수 있게 지으면 좋을거같아요. 

```suggestion
 const LIFE_INFO = [
    '의사소통',
    '위치 및 교통편',
    '가격대비 만족도',
    '청결도',
    '방음',
    '편의시설',
  ];

function LifeInfo() {
  const [open, setOpen] = useState(false);

  return (
    <OpenLayout open={open} setOpen={setOpen} title="생활 정보">
      {LIFE_INFO.map((item, index) => (
        <ScoreInfoField
```

Reply:
해당 컴포넌트가 사용되는 위치가 없어서 우선 삭제했습니다.

https://xd.adobe.com/view/801a8ba1-3bc3-4bd1-8f6a-7c1f85bce740-d549/screen/00885f19-7ce9-4b6a-8838-4ca7f7b85fcc/

디자인도 확인해보니 LIFE_INFO 항목들이 사용되는 부분은 확인하지 못했습니다.